src/styles/style.css

```diff
@@ -0,0 +1,93 @@
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+  list-style-type: none;
+}
+
+body {
+  background: url(../assets/alien.webp);
+  background-color: rgb(116, 114, 112);
+  font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+}
+
+.heading {
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+}
+
+form {
+  width: 500px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 30px 0;
+  box-shadow: 5px 5px #fff, -5px 5px #fff;
+}
+
```

Comment:
- [ ] [REQUIRED] Good job so far! Please keep the general layout of the [wireframe](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/images/leaderboard_wireframe.png), this is the only mandatory requirement.

- ### Your Layout: ❌ 

![image](https://user-images.githubusercontent.com/63915024/182876194-5c08e494-66ad-4c1c-a03c-899f79e40c84.png)


- ### Require Layout: ✔️ 

![leaderboard_wireframe](https://user-images.githubusercontent.com/63915024/182876314-240a3ffe-8335-4a89-8f7b-531c38b33807.png)